pkgs/os-specific/linux/bcc/default.nix

```diff
@@ -29,6 +29,12 @@ python.pkgs.buildPythonApplication rec {
     # This is needed until we fix
     # https://github.com/NixOS/nixpkgs/issues/40427
     ./fix-deadlock-detector-import.patch
+    # Add definition for BTF_KIND_FLOAT, added in Linux 5.14
+    # Can be removed once linuxHeaders (used here via glibc) are bumped to 5.14+.
+    (fetchpatch {
+      url = "https://salsa.debian.org/debian/bpfcc/-/raw/master/debian/patches/0004-compat-defs.patch";
```

Comment:
point to the commit id of current master instead of master, in case debian removes the patch or its content changes? (build will break with checksum, but if we can make build not break instead...)

Reply:
Oops good catch. Obviously should be a permalink.